src/App.tsx

```diff
@@ -1,10 +1,45 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { FilterBy } from './types/Filter';
+import { TodoList } from './components/TodoList/TodoList';
+import { filterTodo } from './components/TodoFilter';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header/Header';
+import { ErrorType } from './types/Errors';
+import { Error } from './components/Error/Error';
 
-const USER_ID = 0;
+const USER_ID = 12106;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorType | null>(null);
+  const [filterBy, setFilterBy] = useState(FilterBy.All);
+
+  const displayError = (error: ErrorType) => {
```

Comment:
This should be implemented in Error component. App should doesnt care about it

Reply:
I know how to do it with context, but how do you suggest it to work without it. If I got what you meant of course